test-utils/mockGenerators/mockUser.js

```diff
@@ -17,8 +17,9 @@ export default function mockUser({
     hasOneNumber: true,
   },
 } = {}) {
-  const firstName = faker.name.firstName();
-  const lastName = faker.name.lastName();
+  // running into duplicate users in e2e 'should be able to register with valid data' test
+  const firstName = faker.name.firstName() + Math.round(Math.random() * 10000);
+  const lastName = faker.name.lastName() + Math.round(Math.random() * 10000);
```

Comment:
Instead of the changes here, maybe try `${Math.round(Math.random() * 10000)}-operationcode.org` for the argument to faker` on L23.

In the meantime, I just wiped the staging DB of all faker emails.